### PR TITLE
User guide update for Apama-ctrl

### PR DESCRIPTION
**Pull request by kailol-sag**
Eliminate wrong Apama-ctrl info from the microservices table and make it simpler as suggested by Rob.
I suggest to backport this back to 10.17. Earlier versions cannot be auto-backported because of the link in the first column: in 10.17, the name of the folder changed from "apama" to "streaming-analytics".

### CODE REVIEW

**Review by rthrippleton — CHANGES_REQUESTED** [review]
(see suggestion)

File: content/users-guide/administration-bundle/managing-microservices.md [hunk 1]
```diff
@@ -57,24 +57,10 @@ Below you find a list of all microservices which are by default subscribed in a
...
-<td style="text-align:left">{{< standard-tenant >}}</td>
-</tr>
-
-<tr>
-<td style="text-align:left"><a href="/streaming-analytics/overview-streaming-analytics/#microservice-and-applications" class="no-ajaxy">Apama-ctrl-smartrulesmt</a></td>
-<td style="text-align:left">Restricted version of the Apama microservice. Runtime for smart rules only, no Analytics Builder models or EPL apps available</td>
-<td style="text-align:left">apama-ctrl-smartrulesmt</td>
-<td style="text-align:left">Only available for self-hosted installations</td>
+<td style="text-align:left"><a href="/streaming-analytics/overview-streaming-analytics/#microservice-and-applications" class="no-ajaxy">Apama-ctrl-*</a></td>
+<td style="text-align:left">Streaming Analytics microservices, including runtime for Analytics Builder, EPL apps and smart rules. Capabilities and resources vary depending on the microservice variant used</td>
```
[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/users-guide/administration-bundle/managing-microservices.md:61]: Looks like a convention (no matter whether I'm happy with that or not ...)

[the remark this answers is not in the document]
Comment (rthrippleton) [reply content/users-guide/administration-bundle/managing-microservices.md:61]: Fair enough!

**Review by rthrippleton — APPROVED** [review]

**Review by skom-sag — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
Chery-picked manually to 10.17 and 10.18 release branches.

**Pull request merged**